cheroot/test/test_core.py

```diff
@@ -262,6 +262,24 @@ def test_content_length_required(test_client):
     assert actual_status == HTTP_LENGTH_REQUIRED
 
 
+@pytest.mark.xfail(reason='https://github.com/cherrypy/cheroot/issues/106')
+def test_large_request(test_client):
+    """Test GET query with maliciously large Content-Length."""
+    # If the server's max_request_body_size is not set (i.e. is set to 0)
+    # then this will result in an Overflow error in the server.
```

Comment:
It's better to spell it exactly as it happens:
```suggestion
    # then this will result in an `OverflowError: Python int too large to convert to C ssize_t`
    # in the server.
```